src/hasReachedStyle.js

```diff
@@ -1,4 +1,6 @@
-export default function hasReachedStyle(currentStyle, style) {
+/* @flow */
+export default function hasReachedStyle(currentStyle: Object,
```

Comment:
I would go for some higher kind of type here instead of Object. I't too generic.